doc/release/upcoming_changes/28343.change.rst

```diff
@@ -0,0 +1 @@
+* In all cases, ``np.linalg.norm``, ``np.linalg.vector_norm``, and ``np.linalg.matrix_norm`` now return zero for arrays with a shape of zero along at least one axis. Previously, NumPy would raises errors or return zero depending on the shape of the array.
```

Comment:
```suggestion
* ``np.linalg.norm``, ``np.linalg.vector_norm``, and ``np.linalg.matrix_norm`` now returns zero for empty arrays for most norms chosen by the ``ord`` parameter (empty arrays have at least one axis of size zero). Previously, NumPy raised an error or returned zero depending on the shape of the array.
```
@ngoldbaum maybe you can have a quick think (suggest-apply-merge)?  Or just merge as is if you prefer...

Actually, maybe the clearest is to just list the changed ones:
```
* The vector norm ``ord=inf`` and the matrix norms ``ord={1, 2, inf, 'nuc'}``
  now always returns zero for empty arrays
  [...]
  This affects `np.linalg.norm`, `np.linalg.vector_norm`, and `np.linalg.matrix_norm`.
```

Reply:
Thanks, I tweaked your suggestion a teeny bit.